app/src/main/java/chat/rocket/android/helper/KeyboardHelper.kt

```diff
@@ -50,4 +51,15 @@ object KeyboardHelper {
             inputMethodManager.toggleSoftInput(InputMethodManager.SHOW_IMPLICIT, InputMethodManager.SHOW_IMPLICIT)
         }
     }
+
+    /**
+     *  Show the soft keyboard for the given view at the start of activity or fragment.
+     *
+     *  @param view View to receive input focus.
+     */
+    fun showSoftKeyboardOnActivityStart(activity: Activity, view: View) {
```

Comment:
The soft keyboard is not showing probably because of this line: https://github.com/RocketChat/Rocket.Chat.Android/blob/45ea38242a208389104c6d9e2baa026df76e71be/app/src/main/AndroidManifest.xml#L55.
Maybe if you remove the `stateAlwaysHidden` flag it will show the keyboard. I think it's better than this approach. :+1: